src/main/java/htsjdk/samtools/util/SortingCollection.java

```diff
@@ -167,18 +171,17 @@ public void add(final T rec) {
             throw new IllegalStateException("Cannot add after calling iterator()");
         }
         if (numRecordsInRam == maxRecordsInRam) {
-            // sample every 100 files written.
+
             long startMem = 0;
-            boolean printRecordSizeSampling = files.size() % 100 == 0 && Log.getGlobalLogLevel() == Log.LogLevel.DEBUG;
```

Comment:
Bike-shedding: why not keep the original name? The new one reads more like an `int` to me.

Reply:
Agreed.